aiven/client/cli.py

```diff
@@ -519,13 +520,63 @@ def service_user_list(self):
         self.print_response(service["users"], format=self.args.format, json=self.args.json,
                             table_layout=layout)
 
+    @arg.project
+    @arg.service_name
+    @arg("--username", help="Service user username", required=True)
+    @arg("-d", "--target-directory", help="Directory to write credentials to", required=False, default=os.getcwd())
+    @arg("-p", "--password", help="Truststore password", default="changeit")
+    def service_user_java_creds(self):
+        """Download user certificate/key/CA certificate and create a Java keystore/truststore/properties from them"""
+        self.service_user_creds_download()
+        # First create the truststore
+        subprocess.check_call([
+            "keytool", "-importcert",
+            "-alias", "Aiven CA",
+            "-keystore", os.path.join(self.args.target_directory, "client.truststore.jks"),
+            "-storepass", self.args.password,
+            "-file", os.path.join(self.args.target_directory, "ca.pem"),
+            "-noprompt",
+        ])
+        # Then create the keystore
+        subprocess.check_call([
+            "openssl", "pkcs12", "-export",
+            "-out", os.path.join(self.args.target_directory, "client.keystore.p12"),
+            "-inkey", os.path.join(self.args.target_directory, "service.key"),
+            "-in", os.path.join(self.args.target_directory, "service.cert"),
+            "-certfile", os.path.join(self.args.target_directory, "ca.pem"),
+            "-passout", "pass:{}".format(self.args.password),
+        ])
+        service = self.client.get_service(project=self.get_project(), service=self.args.name)
+        with open(os.path.join(self.args.target_directory, "client.properties"), "w") as fp:
```

Comment:
This is Kafka specific, should the command name then reflect that this is intended for Kafka?

Reply:
Why is this Kafka specific? Doesn't the same apply for any other service with Java client and client certs, such as Cassandra beta?

Reply:
Cassandra users password authentication, so at least the credentials are different. Truststore is probably generated for server side authentication.

client.properties is surely Kafka specific.

Reply:
Updated name to be Kafka specific.